### PR TITLE
Fix styles on RTL languages.

### LINKED ISSUE #4243: [Devhub Landing] Test and make sure it looks fine in RTL
We didn't test this yet on RTL, let's test it and fix all the things.

### PR DESCRIPTION
**Pull request by EnTeQuAk**
Fixes #4243

screenshot from 2017-01-16 12-14-03

screenshot from 2017-01-13 18-41-26

### CODE REVIEW

**Comment by EnTeQuAk** [comment]
Ah, there's one nit still in the release channel list apparently, see "Spicy Pizza" in the screenshot. I'm fixing this.

**Comment by EnTeQuAk** [comment]
Fixed, this is now ready for review :)

**Comment by atiqueahmedziad** [comment]
I think this image be in left side in the box. 
image

**Comment by EnTeQuAk** [comment]
heh, good point. I'll do that, I think I just forgot to pull the diff from the devtools since it looked alright in the browser 😆

**Comment by EnTeQuAk** [comment]
fixed :)

File: static/css/devhub/new-landing/sections/my-addons.less [hunk 1]
```diff
@@ -101,6 +112,11 @@
 
 .DevHub-MyAddons-item-buttons-submit {
   margin-left: auto;
+
+  html[dir=rtl] & {
+    margin-right: auto;
+    margin-left: 0px;
```
Comment (muffinresearch) [inline static/css/devhub/new-landing/sections/my-addons.less]: You can remove the `px`

**Comment by muffinresearch** [comment]
r+wc

File: static/css/devhub/new-landing/sections/my-addons.less (hunk 1 above)
Comment (EnTeQuAk) [inline static/css/devhub/new-landing/sections/my-addons.less]: fixed, thanks!

**Pull request merged**